java/pom.xml

```diff
@@ -107,6 +111,11 @@
                 <artifactId>constructs</artifactId>
                 <version>${constructs.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.github.cdklabs.cdk_cloudformation</groupId>
+                <artifactId>tf-aws-keypair</artifactId>
+                <version>${awskeypair.version}</version>
+            </dependency>
```

Comment:
Can we update the NOTICES file with this new dependency?

Reply:
Done